### PR TITLE
Speed up list analysis

### PR DESCRIPTION
**Pull request by haneslinger**
#### What's this PR do?

The first commit speeds up the db queries and serialization of the "views" returned by `/api/v3/analyses/` by about 50%
The second makes returning the views optional, as they aren't used. That speed things up _a lot_

#### How should this be manually tested?
1. Create analysis ([as described here](https://github.com/SEED-platform/seed/issues/3343#issue-1296292747)) 
2. Go to property detail page. 
3. Say "wow, so fast."

#### What are the relevant tickets?
https://github.com/SEED-platform/seed/issues/3343

### CODE REVIEW

File: seed/models/analysis_property_views.py [hunk 1]
```diff
@@ -95,19 +95,19 @@ def get_property_views(cls, analysis_property_views):
         property_view_query = models.Q()
         for analysis_property_view in analysis_property_views:
             property_view_query |= (
-                models.Q(property=analysis_property_view.property)
-                & models.Q(cycle=analysis_property_view.cycle)
+                models.Q(property_id=analysis_property_view.property_id)
```
Comment (nllong) [inline seed/models/analysis_property_views.py:98]: I'm pretty sure this is functionally the same, but being explicit that it is the `_id` is nicer.

File: seed/static/seed/js/services/analyses_service.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ angular.module('BE.seed.service.analyses', [])
 
       const get_analyses_for_canonical_property = function (property_id) {
         const org = user_service.get_organization().id;
-        return $http.get('/api/v3/analyses/?organization_id=' + org + '&property_id=' + property_id).then(function (response) {
+        return $http.get('/api/v3/analyses/?&include_views=false&organization_id=' + org + '&property_id=' + property_id).then(function (response) {
```
Comment (nllong) [inline seed/static/seed/js/services/analyses_service.js:35]: 🔥

**Review by nllong — APPROVED** [review]
looks great, the loading of a single property after running 100 properties through better was *much* faster!

File: seed/models/analysis_property_views.py (hunk 1 above)
Comment (haneslinger) [reply seed/models/analysis_property_views.py:98]: I could be wrong, but I think before it was retrieving the whole cycle and property from the db, which cost us an extra query per loop.  

I remember less queries being logged after the change, but there were a lot of logs, so I could be confused.

**Pull request merged**